evm_arithmetization/src/cpu/incr.rs

```diff
@@ -0,0 +1,127 @@
+use plonky2::field::extension::Extendable;
+use plonky2::field::packed::PackedField;
+use plonky2::field::types::Field;
+use plonky2::hash::hash_types::RichField;
+use plonky2::iop::ext_target::ExtensionTarget;
+use plonky2::plonk::circuit_builder::CircuitBuilder;
+use starky::constraint_consumer::{ConstraintConsumer, RecursiveConstraintConsumer};
+
+use super::dup_swap::{
+    channels_equal_ext_circuit, channels_equal_packed, constrain_channel_ext_circuit,
+    constrain_channel_packed,
+};
+use crate::cpu::columns::CpuColumnsView;
+
+/// Evaluates the constraints for the INCR opcodes.
+pub(crate) fn eval_packed<P: PackedField>(
+    lv: &CpuColumnsView<P>,
+    nv: &CpuColumnsView<P>,
+    yield_constr: &mut ConstraintConsumer<P>,
+) {
+    let base_filter = lv.op.incr;
+
+    // Constrain the helper column
+    yield_constr.constraint(
+        base_filter
```

Comment:
I think it's rather `base_filter * (lv.general.incr().is_not_incr1 - lv.opcode_bits[0] * lv.opcode_bits[1])`.

Reply:
This would not evaluate to 0 for INCR2 for instance.